src/Bugsnag/Notifier.cs

```diff
@@ -39,39 +42,71 @@ public void Send(Event errorEvent)
                 Send(notification);
```

Comment:
As an additional feature, we could add a check so if we send a report successfully, we could try to send the stored reports if there are any. Could mitigate a bit against network issues.